it/utils/ItNinoApplicationTestData.scala

```diff
@@ -157,23 +157,23 @@ object ItNinoApplicationTestData {
         "maritalStatus" -> "DIVORCED",
         "startDate" -> writeOrReadDate("1990-10-10", "10-10-1990"),
         "endDate" -> writeOrReadDate(),
-        "partnerNino" -> "AA000000B",
+        "spouseNino" -> "AA000000B",
         "birthDate" -> writeOrReadDate(),
         "forename" -> "Testforename",
         "surname" -> "Testsurname"),
       Json.obj(
         "maritalStatus" -> "DIVORCED",
         "startDate" -> writeOrReadDate("1990-10-10", "10-10-1990"),
         "endDate" -> writeOrReadDate(),
-        "partnerNino" -> "AA000000C",
+        "spouseNino" -> "AA000000C",
         "birthDate" -> writeOrReadDate(),
         "forename" -> "Othertestforename",
         "surname" -> "Testsurname")
     ),
     "originData" -> Json.obj(
       "birthTown" -> "ATown",
       "birthProvince" -> "SomeProvince",
-      "birthCountryCode" -> 200,
+      "birthCountryCode" -> "FR",
```

Comment:
Is this meant to fail? Country codes must be 3 letters long

Reply:
you sure, thought it was up to 3 characters

Reply:
Yes, in the DES spec it specifically says ISO alpha 3 country codes which is a defined list in Appendix A (and also a standard list)